.gitignore

```diff
@@ -11,4 +11,5 @@ out/
 .classpath
 .project
 .settings/
+.sts4-cache/
```

Comment:
Perhaps you should add this to your global .gitignore?

Reply:
It's from the upcoming SpringToolsSuite4. If everybody should have IDE specifc gitignores global, why do we see so many IDE specific project ignores everywhere ... and here.